testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/nuget/NugetITSupport.java

```diff
@@ -60,6 +72,19 @@
   @Inject
   private RepositoryManager repositoryManager;
 
+  @Inject
+  private NugetApiKeyStore keyStore;
+
+  @Inject
+  private RealmManager realmManager;
+
+  @NonNls
+  protected final Logger log = (Logger) Preconditions.checkNotNull(this.createLogger());
```

Comment:
Could move this logger to `NexusCoreITSupport` or even `NexusPaxExamSupport` so everyone benefits